D/DSDP/bundled/patches/makeflags.patch

```diff
@@ -0,0 +1,19 @@
+diff --git before/DSDP5.8/make.include after/DSDP5.8/make.include
+index d1b85f0..0f986de 100644
+--- before/DSDP5.8/make.include
++++ after/DSDP5.8/make.include
+@@ -55,13 +55,12 @@ CLINKER	= ${CC} ${OPTFLAGS}
+ # Not needed to compile library or matlab executable
+ # Needed to link DSDP library to the driver ( read SDPA files, maxcut example, ...)
+ # Also include the math library and other libraries needed to link the BLAS to the C files that call them.
+-LAPACKBLAS  = -llapack -lblas -lg2c -lm
++LAPACKBLAS  = -L${libdir} -lopenblas -lm
```

Comment:
Can't you set this from the command line instead of patching the file?

Reply:
let's try

Reply:
I don't think so because a flag is set by default in the makefile, so I assume it takes precedence over command line?

Reply:
You need to pass variables settings as arguments on the command line:
```
make LAPACKBLAS=....
```